fbw-a32nx/src/systems/instruments/src/PFD/AltitudeIndicator.tsx

```diff
@@ -589,15 +589,15 @@ class AltimeterIndicator extends DisplayComponent<AltimeterIndicatorProps> {
             this.handleBlink();
         });
 
-        sub.on('transAlt').whenChanged().handle((ta) => {
-            this.transAlt = ta;
+        sub.on('fmTransAltRaw').whenChanged().handle((ta) => {
+            this.transAltAr = new Arinc429Word(ta);
```

Comment:
Would be better to use an Arinc429Register so you don't need to allocate a new object every update (applies to all of these words below too).